runhouse/resources/hardware/cluster.py

```diff
@@ -1231,7 +1250,7 @@ def run_python(
         command_str = "; ".join(commands)
         command_str_repr = (
             repr(repr(command_str))[2:-2]
-            if self.ssh_creds.get("password")
+            if self.ssh_creds.values.get("password")
```

Comment:
prob will not need the .values, if self.ssh_creds will return self._ssh_creds.values